paper/joss/reconst_ivim/paper.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Ivim: a model for diffusion and perfusion analysis of MRI'
+tags:
+  - dipy
+  - ivim
+  - diffusion
+  - perfusion
+  - intravoxel incoherent motion
+  - MRI
+  - bi-exponential fitting
+authors:
+ - name: Shahnawaz Ahmed
+   orcid: 0000-0003-1145-7279
+   affiliation: 1
+ - name: Ariel Rokem
+   orcid: 0000-0003-0679-1985
+   affiliation: 2
+ - name: Eric Peterson
+   orcid: 0000-0002-3530-9652
+   affiliation: 3
+ - name: Rafael Neto Henriques
+   orcid: 0000-0000-0000-1234
+   affiliation: 4
+affiliations:
+ - name: Birla Institute of Technology and Science Pilani
+   index: 1
+ - name: The University of Washington eScience Institute
+   index: 2
+ - name: SRI International
+   index: 3
+ - name: Cognition and Brain Sciences Unit (CBU)
+   index: 4
+date: 6 October 2016
+bibliography: paper.bib
+---
+
+# Summary
+
+Intra-voxel incoherent motion (IVIM) is a biophysical model that describes diffusion
```

Comment:
I am being picky here - I will suggest not calling this a biophysical model. One can argue that this is indeed a biophysical model, however I am suggesting to rename this here just to avoid confusions. In general biophysical models only aim to relate the diffusion decay with biological feature (ignoring perfusion). Here the objective is to separate diffusion and perfusion effects. How about replacing the first sentence with the following:

Intra-voxel incoherent motion (IVIM) imaging is a technique that aims to separate effects of perfusion (microcirculation of blood in the capillary network) from the effects of molecular diffusion from the [@le1986mr, @le1988ra].